### PR TITLE
fix(doc): Update useful_scripts_and_tasks.adoc to include lint script

### PR DESCRIPTION
**Pull request by sahil143**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [ ] What does this PR do? 
 - This Pr adds the script to docs which can be used to check for lint errors
Tslint PR: https://github.com/fabric8-ui/fabric8-planner/pull/2673
- [ ] What issue/task does this PR references?

- [ ] Are the tests Included?
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

File: docs/useful_scripts_and_tasks.adoc [hunk 1]
```diff
@@ -40,6 +40,9 @@ v|`npm run build \-- --validate`
 v|`npm run build \-- --watch`
 | Rebuilds planner library on file changes
 
+| lint
+v|`npm run lint-check`
```
Comment (jarifibrahim) [inline docs/useful_scripts_and_tasks.adoc]: This shouldn't be part of `build` section. All the scripts in build section start with `npm run build ...`.

File: docs/useful_scripts_and_tasks.adoc [hunk 2]
```diff
@@ -72,20 +71,25 @@ v|`npm run clean \-- --modules`
 v|`npm run clean \-- --temp`
 | Removes temp files and artifacts
 
+.1+| TsLint
```
Comment (jarifibrahim) [inline docs/useful_scripts_and_tasks.adoc]: ~TsLint~ => TSLint

File: docs/useful_scripts_and_tasks.adoc [hunk 3]
```diff
@@ -72,20 +71,25 @@ v|`npm run clean \-- --modules`
 v|`npm run clean \-- --temp`
 | Removes temp files and artifacts
 
+.1+| TSLint
+
+| lint
```
Comment (jarifibrahim) [inline docs/useful_scripts_and_tasks.adoc]: @sahil143, `lint` isn't a subtask. We should remove it.

**Comment by Preeticp** [comment]
@sahil143, @jarifibrahim  how about: Task: Lint-check, Sub task: add a hyphen(-), Description: Checks for linter error using TSLint

**Comment by sahil143** [comment]
> how about: Task: Lint-check, Sub task: add a hyphen(-), Description: Checks for linter error using TSLint

@Preeticp looks good but there is going to be `npm run lint-fix` also or something similar to this.

**Comment by sahil143** [comment]
@Preeticp I think this is good. Let's go with this one.

**Comment by Raunak1203** [comment]
[test[

**Comment by Raunak1203** [comment]
[test]

**Comment by joshuawilson** [comment]
@jarifibrahim is there something that needs to be done? Or can you approve this?

File: docs/useful_scripts_and_tasks.adoc [hunk 4]
```diff
@@ -72,20 +71,25 @@ v|`npm run clean \-- --modules`
...
+.1+| Tests
 
 | unit
 v|`npm run tests \-- --unit`
 | Runs the unit tests
 //Commented out as not yet implemented
 //| func
 //v|`npm run tests \-- --func`
-| Runs the functional tests
+//| Runs the functional tests
```
Comment (jarifibrahim) [inline docs/useful_scripts_and_tasks.adoc]: Why was this removed?

File: docs/useful_scripts_and_tasks.adoc [hunk 5]
```diff
@@ -72,20 +71,25 @@ v|`npm run clean \-- --modules`
...
 
 //| smok
 //v|`npm run tests \-- --smok`
-| Runs the smoke tests
+//| Runs the smoke tests
```
Comment (jarifibrahim) [inline docs/useful_scripts_and_tasks.adoc]: Why was this removed?

**Comment by jarifibrahim** [comment]
> @jarifibrahim is there something that needs to be done? Or can you approve this?

I've added a small comment. I'll approve this PR once it is addressed.
